tests/integration/conftest.py

```
@@ -0,0 +1,135 @@
+    # Budget: "itest-" (6) + dashes (2) + suffix (6) = 14 chars of overhead
+    # against the 48-char shed-name ceiling, leaving 34 for the prefix.
+    prefix = sanitized[:34].rstrip("-")
+    name = f"itest-{prefix}-{suffix}"
+    yield name
+    # Teardown is best-effort. A test that died mid-create might leave a
+    # half-created shed; ignore_missing=True keeps cleanup from masking
+    # the original failure.
+    try:
+        shed_server.delete(name, ignore_missing=True)
+    except Exception:
+        pass
```

Ruff (0.15.14)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)